app/controllers/properties_controller.rb

```diff
@@ -24,7 +28,7 @@ def show
 
   # GET /properties/new
   def new
-    @property = Property.new
+    @property = Property.new(user_id: current_user.id, status: "pending")
```

Comment:
Might want to update to `current_user.properties.build` for consistency. I think that's a pattern that should be established everywhere its appropriate.